src/components/upcoming/Events.tsx

```diff
@@ -0,0 +1,19 @@
+import { EventType } from '@/lib/types';
+
+import Event from '@/components/upcoming/Event';
+
+import React from 'react';
+
+type Props = { events: EventType[] };
+
+const Events = ({ events }: Props) => {
```

Comment:
Remove this component, just directly include in the page, I think @Deepu178  what do think?

Reply:
Fair enough. Will restructure it ❤️